### ISSUE TITLE
Transfer to Another Planet Issue

### ISSUE THREAD

**Issue by absincog** [MuMech/MechJeb2#738]
Note:  this issue is not corrected as of MJ dev build 595.  Tested just now and the launch window continues to be computed 157 days in the future regardless of the relative orientation of Kerbin and the target planet (Eve, in this case).  

The 'transfer to another planet maneuver' does not create an intercepting trajectory with the target (Eve), requiring a 'fine tune closest approach' maneuver with dv of more than 5,600.  Prior to upgrading to KSP 1.1.2 and MJ 2.5.7, the 'fine tune closest approach' maneuver generally required a dv ranging from <100 to ~300 dv.  I've attached the KSP log --> 

[KSP.log.zip](https://github.com/MuMech/MechJeb2/files/302530/KSP.log.zip)

---------------------------------

I'm having an issue after performing clean installs of KSP 1.1.2 and MJ 2.5.7.
 
Prior to these upgrades, I was working on a mission to Eve. I used MJ repeatedly and reliably to transfer various iterations of my spacecraft to Eve using the 'Launch at interplanetary window',  'Transfer to another planet' and 'Fine tune closest approach to target' maneuvers.  
 
Following the upgrade, after completing the 'Transfer to another planet' maneuver, and attempting to create the 'Fine tune closest approach' maneuver node, I receive a warning that (paraphrasing) the resulting orbit does not approach the target closely. MJ either creates the node with an enormous dV or does not create the node at all.  I also attempted launching at several different points in time over a ~2 year period by accelerating time and then selecting the 'Launch at interplanetary window' check box.  It appears that MJ is always computing the interplanetary launch window to Eve to be 157 days 4 hours 52 minutes in the future, regardless of the relative positions of Kerbin and Eve. 
 
I'm using the same craft and maneuver steps as prior to the software upgrade.

**Comment by sarbian** [comment]
Fixed with the stock 1.1.3 fixes

**Comment by absincog** [comment]
Note - this does not appear to have been fixed.  Just tried a launch at the
interplanetary window to Eve.  MJ set the launch at 176 days out.  Launched
and achieved orbit using Ascent Guidance.  Set an interplanetary transfer
to Eve using the maneuver planner and MJ set the node ~155 days out.  In
the past, when I have launched at the interplanetary window, the
interplanetary transfer node would be set ~1 hour after achieving Kerbin
orbit.  I'll create a new issue.
